doc/mediasdk_release_notes.md

```diff
@@ -0,0 +1,806 @@
+
+**GPU hang reporting procedure:** if GPU hang occurred during HW operation, SDK returns status *MFX_ERR_GPU_HANG* from any *SyncOperation()* call which synchronizes SDK workload affected by hang. In addition SDK rejects to accept new frames and returns *MFX_ERR_GPU_HANG* status from any subsequent call of *EncodeFrameAsync()*, *DecodeFrameAsync()*, *RunFrameVPPAsync()*. It’s available for H.264, H.265, MPEG2-Video decoders and encoders.
+
+**SDK-independent GPU hang reporting procedure:** there is recommended way how to monitor GPU hang incidents independently from SDK. There are the following uevent messages being sent via udev system when GPU hang is happened:
+* “*ERROR=1*” – GPU hang happened
+* “*RESET=1*” – GPU reset is started (GPU is being recovered after GPU hang)
+* “*ERROR=0*” – GPU successfully recovered after GPU hang (reset procedure is completed)
+
+To be able to receive these messages, application need to monitor uevents of drm subsystem (devtype is “drm_minor”):
+```c
+uevent_monitor = udev_monitor_new_from_netlink(udev, "udev");
+udev_monitor_filter_add_match_subsystem_devtype(uevent_monitor, "drm", "drm_minor");
+```
+
+**GPU hang recovery procedure:** it is recommended to process *MFX_ERR_DEVICE_FAILED*, *MFX_ERR_GPU_HANG* and *MFX_ERR_ABORTED* uniformly using the full reset procedure described in “Hardware Device Error Handling” of SDK manual. (I.e. recreate all resources: acceleration device, frames memory, SDK sessions, SDK components).
+
+Informative: usually it takes SDK a few seconds to detect and report GPU hang. During this time all *SyncOperation()* calls for tasks affected by GPU hang will return status *MFX_WRN_IN_EXECUTION*. SDK will report the hang with status *MFX_ERR_GPU_HANG* only after GPU hang is detected and HW recovery mechanism is started by driver.
+
+**Related limitations:**
+* GPU hang reporting is supported for Linux only.
+* In case of multiprocess usage model, SDK may return status *MFX_ERR_GPU_HANG* not to all processes. Application should ensure that upon receiving *MFX_ERR_GPU_HANG* in at least one process, GPU hang recovery procedure will be performed for all processes used the same GPU. To achieve this, application can establish intra process broadcast of *MFX_ERR_GPU_HANG* status returned from SDK session, monitor dmesg or follow “SDK-independent GPU hang reporting procedure” procedure.
+* SDK decoders may buffer decoded frames. Decoder *SyncOperation()* call may return previously buffered frame with *MFX_ERR_NONE* status at the time when GPU hang was already reported by underlying HW for one of next frames. This means that even for synchronous pipeline application may get *MFX_ERR_GPU_HANG* from *DecodeFrameAsync()* earlier than from *SyncOperation()*.
+* False positive GPU hangs (*MFX_ERR_GPU_HANG*) can be reported in a very heavy workloads (CPU and GPU utilization are close to 100%).
+
+## <a id='Package_Contents'>Package Contents</a>
```

Comment:
I think this document is a bad place for this info. It actually misleads since you describe binary package which you provide with each mediasdk release. Likely this is good base for wiki article.